config/nrfconnect/app/check-nrfconnect-version.cmake

```diff
@@ -15,6 +15,13 @@
 #
 
 find_package(Python3 REQUIRED)
+
+# Check nRF Connect SDK version in the CMake configuration phase
+execute_process(
```

Comment:
Does this change belong in the CSG fork here? Or 
Should this be upstreamed into the main repo?

Reply:
I guess that this is already in main Repo, some of the commits were merged (to origin test_event_3.1 brach),  even after code freeze. Because of that, I think that some of the commits are missing onto CSG test_event fork